spec/system/admin/configuration/zones_spec.rb

```diff
@@ -6,12 +6,8 @@
   include AuthenticationHelper
   include WebHelper
 
-  before do
-    login_as_admin
-  end
-
   it "list existing zones" do
-    visit spree.edit_admin_general_settings_path
+    login_as_admin_and_visit spree.edit_admin_general_settings_path
```

Comment:
Nice tip!
Hmm... but are you sure? It looks like they both do the same thing:

https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/support/request/authentication_helper.rb#L13

I'm guessing `login_as` is a Warden helper which may or may not make a request.

Reply:
You are right. I've been thinking of some really old code and that's not relevant any more:

* https://github.com/openfoodfoundation/openfoodnetwork/commit/8b04e45ea5d4c902ada59012f43c8abdd399e308

We can probably DRY all these helpers now.